rebar.config

```diff
@@ -1,4 +1,6 @@
-{deps, [{smpp34pdu, ".*", {git, "https://github.com/bdt-group/smpp34pdu.git", {branch, "master"}}},
+{deps, [{smpp34pdu, ".*", {git,
```

Comment:
Don't forget to revert changes for this line

Reply:
Done.